src/Affects/Connection/resources/config/connection.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the tenancy/tenancy package.
+ *
+ * Copyright Laravel Tenancy & Daniël Klabbers <daniel@klabbers.email>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ *
+ * @see https://tenancy.dev
+ * @see https://github.com/tenancy
+ */
+
+return [
+    /*
+     * The name of the tenant connection, tenancy will create this connection during runtime.
+     */
+    'tenant-connection-name' => env('TENANCY_TENANT_CONNECTION_NAME', 'tenant'),
```

Comment:
I don't think we need this. Can't we move this to the event?

Reply:
How are we reliably going to get this in the env then?
I feel like a simple small config might do better than another event that would set this config.